packages/material-ui-system/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.0.0-alpha.22",
   "private": false,
   "author": "Material-UI Team",
-  "description": "Material-UI System - Design system for Material-UI.",
+  "description": "CSS utilities for rapidly creating custom design.",
```

Comment:
```suggestion
  "description": "CSS utilities.",
```
(Not going to get into that debate again!)

Reply:
It's the tagline we use in https://next.material-ui.com/system/basics/. I don't understand the issue with it. From my perspective, this description is accurate.

For instance, you can either use a `<Card>` if you want to use the premade styles or replace it with a Box if you what something custom. In #24327. @mnajdova Used a card (Card + CardContent) to implement the style of:

<img width="258" alt="Capture d’écran 2021-01-10 à 11 08 24" src="https://user-images.githubusercontent.com/3165635/104120054-2f2fd200-5334-11eb-81d1-e4f9b5f28acc.png">

I have replaced it with a Box, so we don't depend on the styles of the Card that didn't seem to solve a specific UI issue. It's simpler (by definition simpler = fewer things mixed together).

Reply:
Already discussed at length, (I just haven't found the mental energy to rewrite that page a third time), but as agreed at the time,  it's primary use-case is small changes, not custom designs. IF you can find a longer description that doesn't oversell it, that's fine.

Reply:
> it's primary use-case is small changes, not custom designs

We are building most of the branding with the `sx` prop, how about?

```suggestion
  "description": "CSS utilities for rapidly layouting custom design.",
```

Reply:
Good enough